### PR TITLE
Jgfouca/upstream 2021 04 28

### PR DESCRIPTION
**Pull request by jgfouca**
Conflict log:

### both modified:   components/eam/bld/config_files/definition.xml
```
<<<<<<< HEAD
<entry id="shoc_sgs" valid_values="0,1" value="0">
Switch to turn on/off SHOC_SGS package: 0 => no, 1 => yes
</entry>
<entry id="unicon" valid_values="0,1" value="0">
Switch to turn on UNICON package: 0 => off, 1 => on
</entry>
=======
>>>>>>> upstream/master
<entry id="clubb_do_deep" valid_values="0,1" value="0">
```
Fix: Keep shoc_sgs, remove unicon
```
<<<<<<< HEAD
<entry id="pbl" valid_values="uw,hb,hbr,clubb_sgs,shoc_sgs" value="">
=======
<entry id="clubb_single_prec" valid_values="0,1" value="0">
Switch to turn on/off CLUBB single : 0 => double precision, 1 => single precision
</entry>
<entry id="pbl" valid_values="uw,hb,hbr,clubb_sgs" value="">
>>>>>>> upstream/master
```
Fix add shoc_sgs to upstream pbl

### both modified:   components/eam/bld/configure
```
<<<<<<< HEAD
#Set the default microphysics package for CLUBB to mg2
if (defined $opts{'shoc_sgs'}) {
    $microphys_pkg = 'mg2';
}

# But if the physics package is adiabatic, ideal, cam3, cam4, change the default
if ($phys_pkg =~ m/^ideal$|^adiabatic$|^cam[34]$/) {
    $microphys_pkg = 'rk';
}
elsif ($phys_pkg eq 'cam5' and !defined $opts{'clubb_sgs'} and !defined $opts{'shoc_sgs'}) {
=======
# But if the physics package is adiabatic, ideal, change the default
if ($phys_pkg =~ m/^ideal$|^adiabatic$/) {
    $microphys_pkg = 'rk';
} elsif ($phys_pkg eq 'default' and !defined $opts{'clubb_sgs'}) {
>>>>>>> upstream/master
```
Fix: fix comment for mg2, add !defined shoc_sgs to upstream check
```
<<<<<<< HEAD
if ($phys_pkg =~ /cam[34]/) {$macrophys_pkg = 'rk';}
elsif ($phys_pkg =~ /ideal|adiabatic/) {$macrophys_pkg = 'none';}
if ($clubb_sgs == 1) {
  $macrophys_pkg = 'clubb_sgs';
}
if ($shoc_sgs == 1) {
    $macrophys_pkg = 'shoc_sgs';
}
=======
if ($phys_pkg =~ /ideal|adiabatic/) { $macrophys_pkg = 'none'; }
if ($clubb_sgs == 1) { $macrophys_pkg = 'clubb_sgs'; }
>>>>>>> upstream/master
```
Fix: add shoc_sgs check to upstream
```
<<<<<<< HEAD
if ($clubb_sgs == 1) {
  $pbl_pkg = 'clubb_sgs';
}
if ($shoc_sgs == 1){
    $pbl_pkg = 'shoc_sgs';
}
=======

# Override if using CLUBB
if ($clubb_sgs == 1) { $pbl_pkg = 'clubb_sgs'; }
>>>>>>> upstream/master
```
Fix: Add shoc_sgs check to upstream
```
<<<<<<< HEAD
# SHOC
if ($shoc_sgs == 1){
    $cfg_cppdefs .= ' -DSHOC_SGS';
}

# UNICON
if ($unicon) { $cfg_cppdefs .= ' -DUSE_UNICON'; }

=======
>>>>>>> upstream/master
```
Fix: Add shoc_sgs check to upstream

### both modified:   components/eam/bld/namelist_files/namelist_defaults_eam.xml
```
<<<<<<< HEAD:components/eam/bld/namelist_files/namelist_defaults_cam.xml
<ncdata           hgrid="48x96"     nlev="26"              ic_ymd="101" >atm/cam/inic/gaus/cami_0000-01-01_48x96_L26_c091218.nc</ncdata>
<ncdata dyn="eul" hgrid="48x96"     nlev="26"              ic_ymd="901" >atm/cam/inic/gaus/cami_0000-09-01_48x96_L26_c040420.nc</ncdata>
<ncdata           hgrid="48x96"     nlev="30"              ic_ymd="101" >atm/cam/inic/gaus/cami_0000-01-01_48x96_L30_c100426.nc</ncdata>
<ncdata           hgrid="32x64"     nlev="26"              ic_ymd="901" >atm/cam/inic/gaus/cami_0000-09-01_32x64_L26_c030918.nc</ncdata>
<ncdata           hgrid="32x64"     nlev="30"              ic_ymd="101" >atm/cam/inic/gaus/cami_0000-01-01_32x64_L30_c090107.nc</ncdata>
<ncdata           hgrid="8x16"      nlev="26"              ic_ymd="101" >atm/cam/inic/gaus/cami_0000-01-01_8x16_L26_c030228.nc</ncdata>
<ncdata           hgrid="8x16"      nlev="26"              ic_ymd="901" >atm/cam/inic/gaus/cami_0000-09-01_8x16_L26_c030918.nc</ncdata>
<ncdata           hgrid="8x16"      nlev="30"              ic_ymd="101" >atm/cam/inic/gaus/cami_0000-01-01_8x16_L30_c090102.nc</ncdata>

<ncdata dyn="se" hgrid="ne4np4"   nlev="128"            ic_ymd="101" >atm/cam/inic/homme/cami_mam3_Linoz_ne4np4_L128_20200205.nc</ncdata>
=======

>>>>>>> upstream/master:components/eam/bld/namelist_files/namelist_defaults_eam.xml
```
Fix: No SCREAM developer has touched these lines, so we keep upstream (remove everything)
```
<<<<<<< HEAD:components/eam/bld/namelist_files/namelist_defaults_cam.xml
<srf_flux_avg eddy_scheme="SHOC_SGS" >0</srf_flux_avg>
=======
>>>>>>> upstream/master:components/eam/bld/namelist_files/namelist_defaults_eam.xml
```
Fix: Keep SHOC_SGS
```
<<<<<<< HEAD:components/eam/bld/namelist_files/namelist_defaults_cam.xml
<cldfrc_rhminl dyn="fv"          phys="cam5" unicon="1"               > 0.850D0 </cldfrc_rhminl>
<cldfrc_rhminl dyn="se"          phys="cam5" unicon="1"               > 0.920D0 </cldfrc_rhminl>
<cldfrc_rhminl                   phys="cam5" microphys="mg2"          > 0.950D0 </cldfrc_rhminl>
<cldfrc_rhminl                   phys="cam5" microphys="p3"          > 0.950D0 </cldfrc_rhminl>
<cldfrc_rhminl                   phys="cam5"                          > 0.8975D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="1.9x2.5"   phys="cam5" microphys="mg1.5"        > 0.9125D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="1.9x2.5"   phys="cam5"                          > 0.8875D0 </cldfrc_rhminl>

<cldfrc_rhminl hgrid="1.9x2.5"   phys="cam4"                          > 0.910D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="0.9x1.25"  phys="cam4"                          > 0.920D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="0.47x0.63" phys="cam4"                          > 0.920D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="0.47x0.63" phys="cam4" ocn="docn"               > 0.913D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="512x1024"  phys="cam4"                          > 0.903D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="128x256"   phys="cam4"                          > 0.905D0 </cldfrc_rhminl>
<cldfrc_rhminl hgrid="48x96"     phys="cam4"                          > 0.880D0 </cldfrc_rhminl>
<cldfrc_rhminl dyn="se"       phys="cam4"                             > 0.910D0 </cldfrc_rhminl>

=======
<cldfrc_rhminl                   phys="default" microphys="mg2"          > 0.950D0 </cldfrc_rhminl>
>>>>>>> upstream/master:components/eam/bld/namelist_files/namelist_defaults_eam.xml
```
Fix: Keep the 'phys="cam5" microphys="p3"' line, otherwise go with upstream. Not sure about this one because I think cam4/5 support is gone.
```
<<<<<<< HEAD:components/eam/bld/namelist_files/namelist_defaults_cam.xml
<zmconv_c0_lnd                                    phys="cam5"         > 0.0059D0 </zmconv_c0_lnd>
<zmconv_c0_lnd                   hgrid="ne120np4" phys="cam5"         > 0.0035D0 </zmconv_c0_lnd>
<zmconv_c0_lnd  microphys="mg2"  clubb_sgs="1"    phys="cam5"         > 0.0075D0 </zmconv_c0_lnd>
<zmconv_c0_lnd  microphys="p3"  clubb_sgs="1"    phys="cam5"         > 0.0075D0 </zmconv_c0_lnd>
<zmconv_c0_lnd  dyn="fv"                          phys="cam4"         > 0.0035D0 </zmconv_c0_lnd>
<zmconv_c0_lnd  dyn="se"                       phys="cam4"         > 0.0035D0 </zmconv_c0_lnd>
<zmconv_c0_lnd                   hgrid="48x96"    phys="cam4"         > 0.0020D0 </zmconv_c0_lnd>
<zmconv_c0_lnd                   hgrid="128x256"  phys="cam4"         > 0.0040D0 </zmconv_c0_lnd>
<zmconv_c0_lnd                   hgrid="256x512"  phys="cam4"         > 0.0040D0 </zmconv_c0_lnd>
<zmconv_c0_lnd                   hgrid="512x1024" phys="cam4"         > 0.0040D0 </zmconv_c0_lnd>

=======
>>>>>>> upstream/master:components/eam/bld/namelist_files/namelist_defaults_eam.xml
```

Fix: Keep the 'microphys="p3"  clubb_sgs="1"' line, otherwise go with upstream. Not sure about this one because I think cam4/5 support is gone.

### both modified:   components/eam/src/chemistry/utils/modal_aero_calcsize.F90
```
subroutine modal_aero_calcsize_sub(state, deltat, pbuf, ptend, do_adjust_in, &
<<<<<<< HEAD
   do_aitacc_transfer_in, list_idx_in, dgnumdry_m)

=======
   do_aitacc_transfer_in, list_idx_in, update_mmr_in, dgnumdry_m, caller)
>>>>>>> upstream/master
```
Fix: Keep upstream
```
<<<<<<< HEAD
   type(physics_ptend), optional, target, intent(inout) :: ptend       ! indivdual parameterization tendencies
   logical,  optional :: do_adjust_in
   logical,  optional :: do_aitacc_transfer_in
   integer,  optional :: list_idx_in
   real(r8), optional, intent(inout), allocatable, target :: dgnumdry_m(:,:,:) ! interstital aerosol dry number mode radius (m)
=======
   !This subroutine is called from various places in the code
   !"caller" optional variable can hold the name of the subroutine
   !which called this subroutine (for debugging only)
   character(len=*), optional     :: caller
>>>>>>> upstream/master
```
Fix: Keep upstream
```
<<<<<<< HEAD
      if(.not. present(dgnumdry_m)) call endrun('list_idx_in is present but dgnumdry_m is missing '//errmsg(__FILE__,__LINE__))

      if(.not. allocated(dgnumdry_m)) &
           call endrun('list_idx_in is present but dgnumdry_m is not allocated '//errmsg(__FILE__,__LINE__))
=======
      list_idx_local  = list_idx_in
      if (.not. present(dgnumdry_m)) &
           call endrun('list_idx_in is present but dgnumdry_m is missing'//errmsg(__FILE__,__LINE__))

      if(.not. allocated(dgnumdry_m)) &
           call endrun('list_idx_in is present but dgnumdry_m is not allocated'//errmsg(__FILE__,__LINE__))
      dgncur_a => dgnumdry_m(:,:,:)

      !update_mmr should be true ONLY for list_idx = 0
      if(list_idx_local > 0 .and. update_mmr) then
         call endrun('update_mmr should be FLASE for list_idx>0'//errmsg(__FILE__,__LINE__))
      endif
   else
      call pbuf_get_field(pbuf, dgnum_idx, dgncur_a)
>>>>>>> upstream/master
```
Fix: Keep upstream
```
<<<<<<< HEAD
         call endrun('modal_aero_calcsize_diag called with '// &
              'list_idx_in but dgnum_m is not present '//errmsg(__FILE__,__LINE__))
      end if
      if (.not. allocated(dgnum_m)) then
         call endrun('modal_aero_calcsize_diag called with '// &
=======
         call endrun('modal_aero_calcsize_diag called with'// &
              'list_idx_in but dgnum_m is not present '//errmsg(__FILE__,__LINE__))
      end if
      if (.not. allocated(dgnum_m)) then
         call endrun('modal_aero_calcsize_diag called with'// &
>>>>>>> upstream/master
```
Fix: Keep upstream

### both modified:   components/eam/src/chemistry/utils/modal_aero_wateruptake.F90
```
<<<<<<< HEAD
   real(r8), optional, allocatable, target, intent(inout) :: dgnumwet_m(:,:,:)
   real(r8), optional, allocatable, target, intent(inout) :: qaerwat_m(:,:,:)
   real(r8), optional, allocatable, target, intent(inout) :: wetdens_m(:,:,:)
=======
   real(r8), optional, allocatable, target, intent(inout)   :: dgnumwet_m(:,:,:)
   real(r8), optional, allocatable, target, intent(inout)   :: qaerwat_m(:,:,:)
   real(r8), optional, allocatable, target, intent(inout)   :: wetdens_m(:,:,:)
>>>>>>> upstream/master
```
Fix: whitespace change only, keep upstream
```
<<<<<<< HEAD
          .not. present(qaerwat_m)  .or. .not. present(wetdens_m)) then
=======
          .not. present(qaerwat_m)) then
>>>>>>> upstream/master
         call endrun('modal_aero_wateruptake_dr called '// &
              'with list_idx_in but required args not present '//errmsg(__FILE__,__LINE__))
      end if

      ! arrays for diagnostic calculations must be allocated
      if (.not. allocated(dgnumdry_m) .or. .not. allocated(dgnumwet_m) .or. &
<<<<<<< HEAD
          .not. allocated(qaerwat_m)  .or. .not. allocated(wetdens_m)) then
=======
          .not. allocated(qaerwat_m)) then
>>>>>>> upstream/master
```
Fix: Keep upstream

### both modified:   components/eam/src/dynamics/se/se_single_column_mod.F90
```
<<<<<<< HEAD
  if (.not. use_replay .and. get_nstep() .eq. 0 .and. par%dynproc) then
    call cnst_get_ind('NUMLIQ', inumliq, abort=.false.)
    call cnst_get_ind('NUMICE', inumice, abort=.false.)
=======
  if (.not. use_replay .and. get_nstep() .eq. 0) then
    call cnst_get_ind('NUMLIQ', inumliq, abrtf=.false.)
    call cnst_get_ind('NUMICE', inumice, abrtf=.false.)
>>>>>>> upstream/master
```
Fix: PeterB appears to have added the par%dynproc check, so keeping HEAD

### both modified:   components/eam/src/physics/cam/cam_diagnostics.F90
```
<<<<<<< HEAD
   call addfld ('QREFHT',horiz_only,    'A','kg/kg','Reference height humidity')
   call addfld ('WINDSPD_10M',horiz_only,'A','m/s','10m wind speed')
   call addfld ('RHREFHT',horiz_only,    'A','fraction','Reference height relative humidity')
=======
   call addfld ('QREFHT',horiz_only,    'A','kg/kg','Reference height humidity', &
    standard_name = 'specific_humidity')
   call addfld ('U10',horiz_only,    'A','m/s','10m wind speed', &
     standard_name='wind_speed')
   call addfld ('RHREFHT',horiz_only,    'A','1','Reference height relative humidity')
>>>>>>> upstream/master
```
Fix: Keeping the U10->WINDSPD_10M rename that PeterC did, otherwise keeping upstream

### both modified:   components/eam/src/physics/cam/check_energy.F90
```
<<<<<<< HEAD
    call cnst_get_ind('CLDICE', ixcldice, abort=.false.)
    call cnst_get_ind('CLDLIQ', ixcldliq, abort=.false.)
    call cnst_get_ind('RAINQM', ixrain, abort=.false.)
    call cnst_get_ind('SNOWQM', ixsnow, abort=.false.)
    call cnst_get_ind('BVRIM ',  ixrim,  abort=.false.)
=======
    call cnst_get_ind('CLDICE', ixcldice, abrtf=.false.)
    call cnst_get_ind('CLDLIQ', ixcldliq, abrtf=.false.)
    call cnst_get_ind('RAINQM', ixrain, abrtf=.false.)
    call cnst_get_ind('SNOWQM', ixsnow, abrtf=.false.)
>>>>>>> upstream/master
 (this exact same conflict happen twice in this file)
```
Fix: KyleP added a ' ' at the end of BVRIM, upstream removed it entirely. I think taking upstream is the right fix.

### both modified:   components/eam/src/physics/cam/cloud_diagnostics.F90
```
<<<<<<< HEAD
   logical :: do_cld_diag, p3_clouds, mg_clouds, rk_clouds, camrt_rad
=======
   logical :: do_cld_diag, mg_clouds, rk_clouds
>>>>>>> upstream/master
```
Fix: AaronD added p3_clouds, upstream removed camrt_rad; so fix is to add p3_clouds to upstream

### both modified:   components/eam/src/physics/cam/clubb_intr.F90
```
<<<<<<< HEAD

#if defined(CLUBB_SGS) || defined(SHOC_SGS)
=======

#ifdef CLUBB_SGS
>>>>>>> upstream/master
```
Fix: upstream cleaned-up trailing whitespace, HEAD added SOHC_SDS; fix is to clean ws on HEAD

### both modified:   components/eam/src/physics/cam/modal_aer_opt.F90
```
<<<<<<< HEAD
real(r8), allocatable, target :: wetdens_m(:,:,:)  ! aerosol water (g/g) for all modes
!$OMP THREADPRIVATE(dgnumdry_m, dgnumwet_m, qaerwat_m, wetdens_m)
=======
!$OMP THREADPRIVATE(dgnumdry_m, dgnumwet_m, qaerwat_m)
>>>>>>> upstream/master
```
Fix: upstream removed wetdens_m, can't see any change on the SCREAM side so keeping upstream

### both modified:   components/eam/src/physics/cam/phys_control.F90
```
<<<<<<< HEAD
              shallow_scheme .eq. 'CLUBB_SGS' .or. shallow_scheme == 'UNICON'.or. &
              shallow_scheme .eq. 'SHOC_SGS' .or. shallow_scheme.eq.'off')) then
=======
              shallow_scheme .eq. 'CLUBB_SGS' .or. shallow_scheme.eq.'off')) then
>>>>>>> upstream/master
```
Fix: upstream removed UNICON, we added SHOC_SGS, so add SHOC_SDS to upstream

### both modified:   components/eam/src/physics/cam/zm_conv_intr.F90
```
<<<<<<< HEAD
!<songxl 2014-05-20-------------
!  if(trigmem)then
! moist static energy at n-1 time step (J/kg)
    call pbuf_add_field('HU_NM1','global',dtype_r8,(/pcols,pver/), hu_nm1_idx)
! moist convection index at n-1 time step(1=yes, 0=no)
    call pbuf_add_field('CNV_NM1','global',dtype_r8,(/pcols,pver/), cnv_nm1_idx)
! temperature at n-1 time step
    call pbuf_add_field('TM1', 'global', dtype_r8,(/pcols,pver/), tm1_idx)
! specific humidity at n-1 time step
    call pbuf_add_field('QM1', 'global', dtype_r8,(/pcols,pver/), qm1_idx)
!  endif
!>songxl 2014-05-20-------------
=======
! Flux of precipitation from deep convection (kg/m2/s)
   call pbuf_add_field('DP_FLXPRC','global',dtype_r8,(/pcols,pverp/),dp_flxprc_idx)

! Flux of snow from deep convection (kg/m2/s)
   call pbuf_add_field('DP_FLXSNW','global',dtype_r8,(/pcols,pverp/),dp_flxsnw_idx)

! deep gbm cloud liquid water (kg/kg)
   call pbuf_add_field('DP_CLDLIQ','global',dtype_r8,(/pcols,pver/), dp_cldliq_idx)

! deep gbm cloud liquid water (kg/kg)
   call pbuf_add_field('DP_CLDICE','global',dtype_r8,(/pcols,pver/), dp_cldice_idx)

>>>>>>> upstream/master
```
Fix: can't discern any change to this code on the SCREAM side, taking upstream

### both modified:   components/elm/cime_config/config_component.xml
```
<<<<<<< HEAD
      <value compset="1950_EAM%CMIP6-LR.*_ELM">1950_CMIP6LR_control</value>
      <value compset="1950_EAM%CMIP6-HR_ELM">1950_CMIP6HR_control</value>
      <value compset="2010_EAM%CMIP6_ELM">2010_CMIP6_control</value>
      <value compset="2010S_EAM%CMIP6_ELM">2010_CMIP6_control</value>
      <value compset="SSP585_EAM%CMIP6_ELM">2015-2100_SSP585_transient</value>
      <value compset="SSP585_EAM%CMIP6_ELM*_BGC%B">2015-2100_SSP585_CMIP6bgc_transient</value>
      <value compset="2010_EAM%CMIP6-LR_ELM">2010_CMIP6LR_control</value>
      <value compset="2010_EAM%CMIP6-HR_ELM">2010_CMIP6HR_control</value>
      <value compset="2010_EAM.*SCREAM-LR" >2010_defscream_control</value>
      <value compset="2010_EAM.*SCREAM-HR" >2010_defscream_control</value>
      <value compset="2010_EAM.*SCREAM-LR-DYAMOND2" >2010_dy2scream_control</value>
      <value compset="2010_EAM.*SCREAM-HR-DYAMOND2" >2010_dy2scream_control</value>
      <value compset="2000_EAM.*SCREAM-LR" >2000_control</value>
      <value compset="2000_EAM.*SCREAM-HR" >2000_control</value>
=======
      <value compset="1950(?:SOI)?_EAM%CMIP6-LR.*_ELM">1950_CMIP6LR_control</value>
      <value compset="1950(?:SOI)?_EAM%CMIP6-HR_ELM">1950_CMIP6HR_control</value>
      <value compset="2010(?:SOI)?_EAM%CMIP6_ELM">2010_CMIP6_control</value>
      <value compset="2010S.*_EAM%CMIP6_ELM">2010_CMIP6_control</value>
      <value compset="SSP585(?:SOI)?_EAM%CMIP6_ELM">2015-2100_SSP585_transient</value>
      <value compset="SSP585(?:SOI)?_EAM%CMIP6_ELM*_BGC%B">2015-2100_SSP585_CMIP6bgc_transient</value>
      <value compset="2010(?:SOI)?_EAM%CMIP6-LR_ELM">2010_CMIP6LR_control</value>
      <value compset="2010(?:SOI)?_EAM%CMIP6-HR_ELM">2010_CMIP6HR_control</value>
>>>>>>> upstream/master
```
Fix: Add the SOI? change to the SCREAM compsets and add them to upstream

### CODE REVIEW

File: components/eam/bld/namelist_files/namelist_defaults_eam.xml [hunk 1]
```diff
@@ -1243,184 +1055,78 @@
...
 
 <cldfrc_ice            >.false.</cldfrc_ice>
-<cldfrc_ice phys="cam5" >.true.</cldfrc_ice>
+<cldfrc_ice phys="default" >.true.</cldfrc_ice>
 
 <cldfrc_rhminl                                                        > 0.900D0 </cldfrc_rhminl>
-<cldfrc_rhminl dyn="fv"          phys="cam5" unicon="1"               > 0.850D0 </cldfrc_rhminl>
-<cldfrc_rhminl dyn="se"          phys="cam5" unicon="1"               > 0.920D0 </cldfrc_rhminl>
-<cldfrc_rhminl                   phys="cam5" microphys="mg2"          > 0.950D0 </cldfrc_rhminl>
 <cldfrc_rhminl                   phys="cam5" microphys="p3"          > 0.950D0 </cldfrc_rhminl>
```
Comment (bogensch) [inline components/eam/bld/namelist_files/namelist_defaults_eam.xml:1061]: Fairly certain this should be....  phys="default" microphys="p3"

File: components/eam/bld/namelist_files/namelist_defaults_eam.xml [hunk 2]
```diff
@@ -1243,184 +1055,78 @@
...
-<uwshcu_rpen hgrid="1.9x2.5"  microphys="mg1.5"                    >  10.0 </uwshcu_rpen>
 
 <uwshcu_rpen dyn="se"                                           >   5.0 </uwshcu_rpen>
 
 <!-- zm conv -->
 <zmconv_c0_lnd                                                        > 0.0030D0 </zmconv_c0_lnd>
-<zmconv_c0_lnd                                    phys="cam5"         > 0.0059D0 </zmconv_c0_lnd>
-<zmconv_c0_lnd                   hgrid="ne120np4" phys="cam5"         > 0.0035D0 </zmconv_c0_lnd>
-<zmconv_c0_lnd  microphys="mg2"  clubb_sgs="1"    phys="cam5"         > 0.0075D0 </zmconv_c0_lnd>
 <zmconv_c0_lnd  microphys="p3"  clubb_sgs="1"    phys="cam5"         > 0.0075D0 </zmconv_c0_lnd>
```
Comment (bogensch) [inline components/eam/bld/namelist_files/namelist_defaults_eam.xml:1121]: Same, should be phys="default" here

**Review by bogensch — CHANGES_REQUESTED** [review]
Thanks Jim, other than the two easy fixes I found this looks tidy to me (I focused mostly on the SHOC and SCM related conflicts).

**Comment by jgfouca** [comment]
@bogensch , done.

**Review by bogensch — APPROVED** [review]

**Pull request merged**